examples/with-algolia/service/README.md

```diff
@@ -0,0 +1,26 @@
+# Tactical Engineering Serverless Starter
+
+## Getting Started
+
+To start developing locally, ensure you're logged into your Nacelle AWS account. If you are not logged in, or have never set it up, [review the setup docs.](https://nacelle.atlassian.net/wiki/spaces/DD/pages/1146922/Environment+and+Tools+Setup#AWS)
+
+`git clone https://github.com/getnacelle/tactical-engineering-serverless-starter.git` and `git init` to scaffold this repo as a new project.
+
+`npm install` to install dependencies.
+
+`npm run dev` for a live-reloading local dev server.
+
+## Testing
+TODO
+
+## Deploying Changes
+TODO
+
+## Tools Used
+
+- AWS Lambda
+- Severless Framework
+- Typescript (via `serverless-plugin-typescript`)
+- Express.js
+- Jest
+- DataDog
```

Comment:
I recommend updating this README to include context & instructions for the Merchant Developer audience.

Reply:
🤦‍♂️ yep. very good point. :D will fix.